samples/KafkaFlow.Sample.PauseConsumerOnError/PauseConsumerOnExceptionMiddleware.cs

```diff
@@ -21,7 +21,7 @@ public async Task Invoke(IMessageContext context, MiddlewareDelegate next)
         }
         catch (Exception exception)
         {
-            context.ConsumerContext.ShouldStoreOffset = false;
+            context.ConsumerContext.AutoMessageCompletion = false;
```

Comment:
Besides the migration guide that will be done afterward, should the documentation for dynamic workers be included in this PR?

Maybe update this documentations website/docs/guides/consumers/consumers.md#workers

Reply:
I have the documentation implemented in another branch. I'll open a new PR with it.